httplug/usage.rst

```diff
@@ -7,6 +7,6 @@ HTTPlug is relevant for three groups of users:
    :maxdepth: 1
 
    Library users <users>
-   Application developers <application-developers>
+   Application developers <tutorial>
```

Comment:
do you agree with linking to the tutorial instead? i think that is more helpful than the integration part anyways.